packages/mask/src/plugins/VCent/SNSAdaptor/TweetDialog.tsx

```diff
@@ -80,26 +55,40 @@ export default function VCentDialog({ tweetAddress }: { tweetAddress: string })
     const { value: tweets } = useAsync(() => PluginVCentRPC.getTweetData(tweetAddress), [tweetAddress])
     const tweet = first(tweets)
     usePluginWrapper(tweet?.type === 'Offer')
+    const networkDescriptor = useNetworkDescriptor(NetworkPluginID.PLUGIN_EVM, ChainId.Mainnet)
```

Comment:
You can always use a `<PluginIDContextProvider />` and `<PluginWeb3ContextProvider />` if the plugin serves for a specific network. Please take some time to learn it: https://docs.mask.io/docs/web3/hooks.